src/Standard.Licensing/LicenseAttributes.cs

```diff
@@ -74,20 +76,22 @@ public virtual void AddAll(IDictionary<string, string> features)
         /// <param name="key">The key of the element.</param>
         public virtual void Remove(string key)
         {
-            var element =
-                xmlData.Elements(childName)
-                    .FirstOrDefault(e => e.Attribute("name") != null && e.Attribute("name").Value == key);
+            XElement element =
+                XmlData.Elements(ChildName)
+                    .FirstOrDefault(e => e.Attribute("name") != null && e.Attribute("name")?.Value == key);
```

Comment:
Change from var, but also there isn't a need for the `?` here since we just performed a null check in the first part of the statement. This could probably be improved to not pull the name attribute twice, so maybe examine that approach.